pkgs/by-name/atomic-server/package.nix

```diff
@@ -12,12 +13,22 @@ in
     pname = "atomic-server";
     version = "0.37.0";
 
-    src = fetchCrate {
-      inherit pname version;
-      hash = "sha256-/OKYac0HA9EWDQ5qNyMPITN5iUdLM9SAVmOm6PVIFOk=";
-    };
+    src = fetchFromGitHub {
+      owner = "atomicdata-dev";
+      repo = pname;
+      rev = "v${version}";
+      hash = "sha256-+Lk2MvkTj+B+G6cNbWAbPrN5ECiyMJ4HSiiLzBLd74g=";
+     };
+
+    cargoHash = "sha256-cSv1XnuzL5PxVOTAUiyiQsMHSRUMaFDkW2/4Bt75G9o=";
 
-    cargoHash = "sha256-LwSyK/7EEoTf1x7KGtebPxYTqH3SCjXGONNMxcmdEv0=";
+    # server/assets_tmp is the directory atomic-server's build will check for
+    # compiled frontend assets to decide whether to rebuild or not
+    # https://github.com/atomicdata-dev/atomic-server/blob/ba3c5959867a563d4da00bb23fd13e45e69dc5d7/server/build.rs#L22-L37
+    postUnpack = ''
+      mkdir -p $PWD/source/server/assets_tmp
+      cp -r ${atomic-browser}/* $PWD/source/server/assets_tmp
```

Comment:
both $PWD and $TMP seemed to work here, which one or is another option recommended?

Reply:
You could even do:

```suggestion
      mkdir -p source/server/assets_tmp
      cp -r ${atomic-browser}/* source/server/assets_tmp
```

Reply:
🤦 of course a relative path would implicitly point to `$PWD` by default. Verified your change still builds, if it's the favored style I can switch to it

Reply:
IIUC this is the common thing to do.

Reply:
thanks, updated!